jenkins-local-terraform/README.md

```diff
@@ -0,0 +1,14 @@
+# Local Jenkins with Terraform and Docker Desktop
+
```

Comment:
A list of resources created by the terraform configuration would be a good addition to this file.